### PR TITLE
Tidying up PPP

### PR DESCRIPTION
**Pull request by HynRidge**

### CODE REVIEW

File: docs/team/hynridge.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Given below are my contributions to the project.
     [3](https://github.com/HynRidge/ped/issues/3),
     [4](https://github.com/HynRidge/ped/issues/2))
     
-## \[Optional\] Contributions to the User Guide  (Extract)
+## Contributions to the User Guide  (Extract)
```
Comment (Ma-Yueran) [inline docs/team/hynridge.md:60]: Should we remove "Optional" here?

Comment (HynRidge) [reply docs/team/hynridge.md:60]: I think..if not it looks weird @Ma-Yueran

Comment (Ma-Yueran) [reply docs/team/hynridge.md:60]: ok, I will change too

**Review by Ma-Yueran — APPROVED** [review]
LGTM!

**Pull request merged**